### ISSUE TITLE
video is not streaming

### ISSUE THREAD

**Issue by LeeChanWu** [xindawndev/LiveScreenStreamForiOS#7]
I have tested this sdk using iphone6(11.3).
Audio is streaming, but video is not streaming.
What is wrong?

**Comment by xindawndev** [comment]
Please contact us by email

2018-06-05 22:15 GMT+08:00 LeeChanWu <notifications@github.com>:

**Comment by LeeChanWu** [comment]
Hello, xindawndev

I have tried to test with your sdk in phone6+(11.3).
Audio is streaming, but video is not streaming.
What is wrong?

Regards

**Comment by xindawndev** [comment]
the ios 11 device doesn't automatically connect to airplay,
swipe up from the bottom of you screen to access control center,
and tap the name of the XBMC-GAMEBOX(XinDawn), Recording Screen will start.



2018-06-05 23:21 GMT+08:00 LeeChanWu <notifications@github.com>:

**Comment by kklwg** [comment]
yes!  I also view this issue. Can you tell me how to resolve it？ （The video can't stream）
